homeworks/react-basics/horbyshyn/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './App.css';
+const axios = require('axios');
+class LecturesData extends React.Component{
+  state = {
+    lectures: []
+  }
+  componentDidMount() {
+    axios.get('https://raw.githubusercontent.com/kottans/chernivtsi-frontend-2019/master/homeworks/react-basics/data.json')
+      .then(res => {
+        const lectures = Object.values(res.data.lectures);
+        this.setState({ lectures });
+      })
+  }
+  
+  render(){
+    return (
+      <div className="lectures">
+      {this.state.lectures.map((item, i) => (
+          RenderLecture(item, i)
+      ))}
+      </div>
+    );
+  }
+}
+
+function RenderLecture(item, key){
+  function handleClick(index) {
+    console.log(index);
+    var id = "lecture-"+index;
+    console.log(document.getElementById(id).classList);
+    if(document.getElementById(id).classList.contains('hidden'))
```

Comment:
Transform `Render Lecture` to class component and use state `hidden`, then you can handle state `onClick()` and render Lecture info only if `hidden` is `false`. You have to control logic in react using state, using dom will cause problem with rerender and will cause conflicts with few components with the same indexes.